Update set-vmnetworkadaptervlan.md (#1034)
MicrosoftDocs/windows-powershell-docs: docset/windows/hyper-v/set-vmnetworkadaptervlan.md

Gets virtual machine Redmond and sets the virtual network adapters in the virtual machine to the untagged mode.

### Example 5
```
PS C:\> Set-VMNetworkAdapterVlan -ManagementOS -Access -VlanID 20
```

Sets the virtual switch in the management OS to the Access mode.
Traffic sent by this virtual switch is tagged with VLAN ID 20.

## PARAMETERS

### -Access

> Specifies **Access** mode for the virtual machine network adapter. This parameter configures an untagged virtual port with the VLANId (port based VLAN), for this reason must be specified in conjunction with parameter **VlanId**.

Reviewer: ```suggestion
Specifies **Access** mode for the virtual machine network adapter. This parameter configures an untagged virtual port with the VLANId (port based VLAN), so it must be specified together with **VlanId**.
```

Author: I am not sure about this... Maybe:

"..., for this reason must be specified together with the **VlanId** parameter."

What do you think @get-itips @tseward @nenonix?

Reviewer 2: Both are ok I think

```

### -Trunk

> Specifies **Trunk** mode for the virtual machine network adapter. This parameter configures a tagged virtual port that passes all allowed VLANId tags to the VM adapter, traffic with the NativeVLANId is passed untagged to the VM adapter.

Reviewer: ```suggestion
Specifies **Trunk** mode for the virtual machine network adapter. This parameter configures a tagged virtual port that passes all allowed VLANId tags to the VM adapter. Traffic with the NativeVLANId is passed untagged to the VM adapter.
```

Reviewer 3: agree
